app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -235,24 +213,45 @@ a.inplace-editing--trigger-link,
   border-width: 1px
   overflow: visible
 
-.inplace-edit--control
-  background: $inplace-edit--color--very-dark
-  width: 1.5rem
-  height: 1.5rem
-  line-height: 1.5rem
-  border: 1px solid $inplace-edit--dark-background
+
+.inplace-edit--controls
   display: inline-block
-  border-radius: 0.75rem
-  font-size: 0.75rem
+  background: $inplace-edit--dark-background
+  border: 1px solid $inplace-edit--color--very-dark
+  box-shadow: 2px 2px 4px $inplace-edit--border-color
+  text-align: center
+  float: right
+  padding: 5px
+  line-height: 1.6rem
+  margin-top: -1px
+  // Having to get the element out of the normal dom flow to
+  // prevent the element overlapping and thereby blocking the click on
+  // inplace-editable elements below the current one.
+  position: absolute
+  right: 0
 
-  &.-icons-2
-    width: 2.875rem
+
+.inplace-edit--control
+  width: 1.75rem
+  height: 1.75rem
+  line-height: 1
```

Comment:
I think the icons are not in the vertical middle...
When you use the same height value for the line-height (1.75rem) and remove padding bottom and top from the inside a tag, the icon will be vertically centered.